cfgov/cfgov/settings/local.py

```diff
@@ -19,7 +19,7 @@
                 'PASSWORD': os.environ.get('MYSQL_PW', ''),
                 'HOST': os.environ.get('MYSQL_HOST', ''),
                 'PORT': os.environ.get('MYSQL_PORT', ''),
-                'OPTIONS': {'init_command': os.environ.get('STORAGE_ENGINE', 'SET storage_engine=MYISAM') },
```

Comment:
Our prod servers are still running MySQL 5.1 at the moment, and 'storage_engine' still seems like the correct syntax there? I'll experiment to see if SET default_storage_engine works on 5.1 though